src/OpenTelemetry/README.md

```diff
@@ -308,6 +306,21 @@ Core, Console.
 }
 ```
 
+Tip: You can find the "current working directory" by calling
+[GetCurrentDirectory](https://docs.microsoft.com/dotnet/api/system.io.directory.getcurrentdirectory)
+method from your code.
+
+As an example, for AspNetCore example project,
```

Comment:
please add link to aspnetcore example.

Reply:
I tried to link to the directory or the csproj file, but seems docfx doesn't allow that. Only *.cs and *.md file is allowed.

Reply:
I'll create a README.md file for the link.

Reply:
Done.